eng/CodeChecks.ps1

```diff
@@ -71,7 +76,7 @@ try {
     if ($LastExitCode -ne 0) {
         $status = git status -s | Out-String
         $status = $status -replace "`n","`n    "
-        LogError "Generated code is not up to date. You may need to run eng\Update-Snippets.ps1 or sdk\storage\generate.ps1"
+        LogError "Generated code is not up to date. You may need to run eng\Update-Snippets.ps1 or sdk\storage\generate.ps1 or Generate-API.ps1"
```

Comment:
Nit: use commas between articles, i.e. "eng\Update-Snippets.ps1, sdk\storage\generate.ps1, or Generate-API.ps1"

But what's the path for Generate-API? Also, that's a non-standard verb. Probably won't make a difference depending on strictness of script hosts, but some will throw when importing. Might use Extract-API or something instead.

Reply:
I'm fine with Extract-API